source/install/self-managed-changelog.md

```diff
@@ -6,12 +6,118 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v7.9 - Feature Release](#release-v7-9-feature-release)
 - [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
 - [Release v7.7 - Feature Release](#release-v7-7-feature-release)
-- [Release v7.5 - Feature Release](#release-v7-5-feature-release)
-- [Release v7.4 - Feature Release](#release-v7-4-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
 
+## Release v7.9 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: March 16th, 2023**
+
+### Important Upgrade Notes
+
+ - Added a new API endpoint ``GET api/v4/posts/[POST_ID]/edit_history``. Users can run "CREATE INDEX CONCURRENTLY" before doing the upgrade to allow PostgreSQL to create the index without locking the table.
```

Comment:
I just updated that PR description with times for the index creation. Can we use this?

>Added a new API endpoint `GET api/v4/posts/[POST_ID]/edit_history`.
>Added a new index on `Posts(OriginalId)`. For a database with 11.8 million posts, on a machine with a i7-11800H CPU (8 cores, 16 threads), 32GiB of RAM and SSD, the index creation takes:
>- MySQL: 98.51s
>- PostgreSQL: 2.6s
>
> To avoid any table locking, servers using Postgres are recommended to create the index online before performing the upgrade using the `CONCURRENTLY` option, as in 
>```CREATE INDEX CONCURRENTLY idx_posts_original_id ON Posts(OriginalId);```